docs/source/index.rst

```diff
@@ -0,0 +1,53 @@
+ALFASIM-SDK
+===========
+
+The ALFAsim-SDK help developers to create plugins for an ALFAsim application.
```

Comment:
Padronize all occurrences of "alfasim-sdk", the initial page itself has 3 different writings (ALFASIM-SDK at the title, ALFAsim-SDK at first paragraph and then alfasim-SDK in the text).